core-feature-pack/pom.xml

```diff
@@ -112,6 +112,11 @@
 
         <dependency>
             <groupId>org.jboss.logging</groupId>
+            <artifactId>jboss-logging-annotations</artifactId>
```

Comment:
@dmlloyd Do you remember why this came in like this?  Do you see a problem if I switch to provided as in Kabir's example?